src/shared/logs-show.c

```diff
@@ -1073,7 +1073,9 @@ int add_matches_for_unit(sd_journal *j, const char *unit) {
         );
 
         if (r == 0 && endswith(unit, ".slice")) {
-                char *m5 = strappend("_SYSTEMD_SLICE=", unit);
+                char *m5;
```

Comment:
Actually I changed that code in one of the recents PRs, in an almost identical way, so there'll be a conflict. It did look familiar.